docs/tutorial/using-checkbox/checkbox-commands.rst

```diff
@@ -0,0 +1,296 @@
+    -----------[ Collect information about installed software packages ]------------
+    ID: com.canonical.certification::package
+    Category: com.canonical.plainbox::uncategorised
+    ... 8< -------------------------------------------------------------------------
+    name: accountsservice
+    version: 22.07.5-2ubuntu1.4
+
+    (...)
+
+    ==============[ Running job 3 / 3. Estimated time left: 0:00:01 ]===============
+    -----------------------------[ audio/list_devices ]-----------------------------
+    ID: com.canonical.certification::audio/list_devices
+    Category: com.canonical.plainbox::audio
+    ... 8< -------------------------------------------------------------------------
+     0 [PCH            ]: HDA-Intel - HDA Intel PCH
+                          HDA Intel PCH at 0xf7f10000 irq 31
+     1 [H340           ]: USB-Audio - Logi USB Headset H340
+                          Logitech Inc. Logi USB Headset H340 at usb-0000:00:14.0-1, full speed
+     2 [HDMI           ]: HDA-Intel - HDA ATI HDMI
+                          HDA ATI HDMI at 0xf7e60000 irq 32
+     3 [U0x46d0x802    ]: USB-Audio - USB Device 0x46d:0x802
+                          USB Device 0x46d:0x802 at usb-0000:00:14.0-4, high speed
+    ------------------------------------------------------------------------- >8 ---
+    Outcome: job passed
+    Finalizing session that hasn't been submitted anywhere: checkbox-run-2023-07-24T09.01.24
+    ==================================[ Results ]===================================
+     ☑ : Collect information about installed software packages
+     ☑ : Collect information about hardware devices (udev)
+     ☑ : audio/list_devices
+
+A few things to notice:
+
+- Checkbox has executed the ``com.canonical.certification::audio/list_devices``
+  job. Because it understands regular expression patterns, you were able to
+  pass it ``.*audio/list_devices`` instead of the full job id.
+- The output is quite similar to running Checkbox manually, except only a text
+  summary is generated at the end (no submission files, no request to upload
+  results to the Certification website).
+- Although you asked Checkbox to run one job, it actually ran three. This is
+  because the ``audio/list_devices`` job definition has some requirements from
+  other jobs (namely, ``com.canonical.certification::packages`` and
+  ``com.canonical.certification::device``), so these jobs are executed as well
+  when you run it.
+
+You can execute a set of jobs by using an appropriate regular expression. For
+instance, ``checkbox.checkbox-cli run .*audio/.*`` would run every job whose
+``id`` contain the string ``audio/`` (as well as the jobs they depend on).
+
+.. warning::
+
+    Be careful when using the ``run`` command with such open regular
+    expressions because you might end up running quite a lot of jobs!
+
+You can also run a whole test plan using the ``run`` command:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli run TODO
```

Comment:
hopefully if https://github.com/canonical/checkbox/pull/567 is merged, we can mention that run takes also a launcher arg (useful for manifest or environment)